src/main/java/com/spotify/reaper/ReaperApplication.java

```diff
@@ -64,25 +69,44 @@
 public class ReaperApplication extends Application<ReaperApplicationConfiguration> {
 
   static final Logger LOG = LoggerFactory.getLogger(ReaperApplication.class);
-
+  public static final UUID REAPER_INSTANCE_ID = UUID.randomUUID();
+  private static String reaperInstanceAddress;
+  private final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+  private static final String DEFAULT_INSTANCE_ADDRESS = "127.0.0.1";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this IP "127.0.0.1" address configurable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1313)